### ISSUE TITLE
WCAG 4.1.2: Form elements must have labels (.form-control.input[placeholder=""])

### ISSUE THREAD

**Issue by zcjess** [zcjess/Spire#5]
#### Issue
`Form elements must have labels` ([label](https://accessibilityinsights.io/info-examples/web/label))
#### Target application
[Opportunities - Spire](https://staging.spirelearn.com/programs/1/opportunities)
#### Element path
div:nth-child(1) > .formkit-outer[data-type="FormKitDatePicker"] > .direct-calendar.formkit-wrapper > .formkit-inner > .datepicker_spire[data-type="FormKitDatePicker"] > .form-control.input[placeholder=""]
#### Snippet
`<input class=" form-control input" placeholder="" tabindex="0" type="text" readonly="readonly">`

Fix ONE of the following:
Form element does not have an implicit (wrapped) <label>
Form element does not have an explicit <label>
aria-label attribute does not exist or is empty
aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
Element has no title attribute
Element has an empty placeholder attribute
Element's default semantics were not overridden with role="none" or role="presentation"
![image](https://github.com/zcjess/Spire/assets/90011962/db782f99-b182-4077-8d91-315bf494ca22)
![image](https://github.com/zcjess/Spire/assets/90011962/c8900264-73ab-4dcb-b320-e895d0e61645)
